common/pdf-jpg-export-js/PdfExportJS/Pages/_Host.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@page "/"
+@namespace PdfExportJS.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PdfExportJS</title>
+    <base href="~/" />
```

Comment:
```suggestion
<base href="/" />
```

Reply:
@ivanchev Per the .NET templates

.html uses "/" (wasm client)

`    <base href="~/" />`

.cshtml uses "~/" (server)

`    <base href="~/" />`